src/ui/templates/results/resultsheader.hbs

```diff
@@ -14,11 +14,15 @@
   {{#if (and _config.showResultCount resultsCount)}}
     <div class="yxt-ResultsHeader-resultsCount" aria-label="{{resultsCountStart}} through {{resultsCountEnd}} of {{resultsCount}}">
       <div aria-hidden="true">
-        <span class="yxt-ResultsHeader-resultsCountStart">{{resultsCountStart}}</span>
-        <span class="yxt-ResultsHeader-resultsCountDash">-</span>
-        <span class="yxt-ResultsHeader-resultsCountEnd">{{resultsCountEnd}}</span>
-        <span class="yxt-ResultsHeader-resultsCountOf">of</span>
-        <span class="yxt-ResultsHeader-resultsCountTotal">{{resultsCount}}</span>
+        {{#if _config.resultsCountTemplate}}
```

Comment:
I thought about whether we want to override the entire "yxt-ResultsHeader-resultsCount" div or just this portion, but I think the use case here is mostly for overriding the text. We probably still want to let them add arbitrary html in case they have some specific styling in mind.

Reply:
@tmeyer2115 convinced me that we don't want to let them add random html here (also confirmed with rose)